File: cms_server/cms_server/Entities/HospPatientEntity.cs
using System;

namespace cms_server.Entities
{
    public class HospPatientEntity
    {
        public string hospital_no { get; set; }
        public string user_pk { get; set; }
        public string pic_dest { get; set; }
        public string prefix { get; set; }
        public string first_name { get; set; }
        public string middle_name { get; set; }
        public string last_name { get; set; }
        public string suffix { get; set; }
        public string gender { get; set; }
        public DateTime? birth_date { get; set; }
        public string birth_place { get; set; }
        public string cs_pk { get; set; }
        public string nat_pk { get; set; }
        public string rel_pk { get; set; }
        public string email { get; set; }
        public string mob_no { get; set; }
        public string line1 { get; set; }
        public string line2 { get; set; }
        public int? brgy_pk { get; set; }
        public int? citymun_pk { get; set; }
        public int? prov_pk { get; set; }
        public string region_pk { get; set; }
        public string zip_code { get; set; }
        public int? consult_count { get; set; }
        public DateTime? last_consult_at { get; set; }
        public DateTime? last_updated_at { get; set; }
        public string last_updated_by { get; set; }

        //props as params
        public string consult_req_pk { get; set; }


    }
}

File: cms_web/src/Services/Entities/HospPatientEntity.ts
interface HospPatientEntity {
  hospital_no?: string;
  user_pk?: string;
  pic_dest?: string;
  prefix?: string;
  first_name?: string;
  middle_name?: string;
  last_name?: string;
  birth_date?: string | Date;
  birth_place?: string;
  cs_pk?: number;
  nat_pk?: number;
  rel_pk?: number;
  email?: string;
  mob_no?: string;
  line1?: string;
  line2?: string;
  brgy_pk?: number;
  citymun_pk?: number;
  prov_pk?: number;
  region_pk?: string;
  zip_code?: string;
  consult_count?: number;
  last_consult_at?: string | Date;
  last_updated_at?: string | Date;
  last_updated_by?: string;

  //props as params
  consult_req_pk?: string;
}

export interface HospPatientTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<HospPatientEntity>;
}

export default HospPatientEntity;
